hub/policy/src/test/java/uk/gov/ida/hub/policy/domain/controller/EidasCountrySelectedStateControllerTest.java

```diff
@@ -63,6 +63,7 @@
     private static final String SELECTED_COUNTRY = STUB_COUNTRY_ONE;
     private static final String ANALYTICS_SESSION_ID = "some-session-id";
     private static final String JOURNEY_TYPE = "some-journey-type";
+    private static final Optional<CountrySignedResponseContainer> EMPTY_COUNTRY_SIGNED_RESPONSE = Optional.empty();
```

Comment:
The other params throughout this file just use `Optional.empty()` in place. I actually prefer what you've done here as it's much clearer what the parameter is for. Perhaps we should be consistent and do it for all the params?

Reply:
Agreed, have updated the rest of this file. I suspect there are other places that this could be done across the test suite, but that's a job for another day...